spec/outputs/webhdfs_spec.rb

```diff
@@ -0,0 +1,142 @@
+# encoding: utf-8
+require 'logstash/devutils/rspec/spec_helper'
+require 'logstash/outputs/webhdfs'
+require 'webhdfs'
+require 'json'
+
+describe 'outputs/webhdfs' do
```

Comment:
can you add here, an integration flag also?, you can see how it can be done by taking a look at https://github.com/logstash-plugins/logstash-input-s3/blob/master/spec/integration/s3_spec.rb#L7